kafka/testhelpers.go

```diff
@@ -52,26 +53,31 @@ var defaulttestconfConfig = [1]string{"api.version.request=true"}
 
 const defaulttestconfBrokers = "localhost:9092"
 
+// flag for semaphore job
+var semaphoreJob = flag.Bool("onSemaphore", false, "Tells if the job is running on Semaphore")
```

Comment:
Rename the flag to clients.semaphore

That's the naming convention we've chosen for flags, for example, testify flags are called "testify.<flag>" and so on.

Reply:
done !